src/components/NavBar.tsx

```diff
@@ -1,11 +1,20 @@
-import { Link } from "react-router-dom";
+import { Link, useLocation } from "react-router-dom";
 import { useState } from "react";
 import "../styles/NavBar.scss";
 import React from "react";
 
 export function NavBar() {
+
+    const location = useLocation();
+    const removedSlash = location.pathname.slice(1);
+    let pageName = removedSlash.substr(0, 1).toUpperCase() + removedSlash.substr(1).toLowerCase();
+
+    if (pageName === "") {
+        pageName = "Home";
+    }
```

Comment:
We could do this all with a ternary operator
```suggestion
    const pageName = removedSlash === ""
        ? "Home"
        : removedSlash.substr(0, 1).toUpperCase() + removedSlash.substr(1).toLowerCase();
```